Sources/Plasma/NucleusLib/pnInputCore/plInputMap.h

```diff
@@ -47,14 +47,16 @@ You can contact Cyan Worlds, Inc. by email legal@cyan.com
 #include "HeadSpin.h"
 #include "plControlDefinition.h"
 
+#include <map>
+#include <string_theory/string>
```

Comment:
We don't actually have any `ST::string` members here, so a forward declaration is sufficient.